pybikes/data/hourbike.json

```diff
@@ -0,0 +1,50 @@
+{
+    "instances": [
+        {
+            "tag": "readybike",
+            "feed_url": "https://www.readybike.co.uk/map-of-stations.html",
```

Comment:
url does not exists anymore

Reply:
also see: https://road.cc/content/news/218094-reading-bike-hire-schemes-future-chaos